src/utils/blockchain/requests/staking.ts

```diff
@@ -32,24 +32,32 @@ export function staking_withdrawFee(
   tokenAddress: string,
   processedCheckpoints: string,
   account: string,
+  useNewContract = false,
 ) {
-  return contractWriter.send('feeSharingProxy_old', 'withdraw', [
-    tokenAddress,
-    processedCheckpoints,
-    account,
-  ]);
+  return contractWriter.send(
+    useNewContract ? 'feeSharingProxy' : 'feeSharingProxy_old',
```

Comment:
This condition is used on 4 places and it could get tricky if contract name changes in the future. Extract it to a method (e.g. getFeeSharingProxyContractName) that will accept 1 parameter (`useNewContract`) and call the method in all 4 places (this file + `useStaking_getAccumulatedFees.ts`).